src/components/career-form/career-file-upload-type.tsx

```diff
@@ -29,40 +40,159 @@ export const CareerFileUploadType = ({
 }: CareerFileUploadTypeProps) => {
   const fileName = file.name.split('.');
   const name = `category_select.${fileName[0]}`;
-  const state = watch(name);
+  const state = watch(name, 'default');
```

Comment:
`state` is quite general. maybe something like `selectedFileType`?

Reply:
I can only second this, makes it much more readable.